src/actions/dispute.js

```diff
@@ -3,7 +3,10 @@ import { createActions } from 'lessdux'
 /* Actions */
 
 // Disputes
-export const disputes = createActions('DISPUTES')
+export const disputes = {
+  ...createActions('DISPUTES', { withUpdate: true }),
+  FETCH_DEADLINES: 'FETCH_DISPUTE_DEADLINES'
```

Comment:
Why aren't deadlines a new resource?

Reply:
To me it makes sense to have `disputes` be updatable and load them separately as part of the same resource. That way when we have implemented a longer term solution to performance issues with event log based data we can simply add the call back into `fetch`. Also it made things simpler so that the rendering doesn't have to map each deadline to the correct dispute.